azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+# NOTE: this Azure configuration is used only for building the sdist and wheels.
+# It uses the OpenAstronomy azure template described in detail at
+# https://openastronomy-azure-pipelines.readthedocs.io/en/latest/
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: astropy
+    name: OpenAstronomy/azure-pipelines-templates
+    ref: master
+
+# NOTE: for now we only use Azure Pipelines on v* branches and tags
+# for building the source and wheel distributions. If you want to
+# make changes to this configuration via a pull request, you can
+# *temporarily* change the branches include trigger to just '*'
+trigger:
+  branches:
+    include:
+    #- 'v*'
```

Comment:
Do we really want the trigger to happen on all v* branches? In the run-up to releases on LTS branches there's often a few commits for things like cleaning up changelogs, etc, which shouldn't get built, right?

Reply:
To be clear they will be built but not uploaded - I think having them build on any push to a v* branch is a good idea as it gives advance indication of whether the wheels are broken, as opposed to just at the point of tagging. It's a compromise since at least we aren't building in master and all PRs.

Reply:
Gotcha, understood, that sounds good then

Reply:
Gotcha, that sounds good then.